graphql/openreader/src/opencrud/orderBy.ts

```diff
@@ -43,8 +43,9 @@ function buildOrderByMapping(model: Model, typeName: string, depth: number): Ope
                     m.set(key + '_ASC', { [key]: 'ASC' })
                     m.set(key + '_DESC', { [key]: 'DESC' })
                     m.set(key + '_ASC_NULLS_FIRST', { [key]: 'ASC NULLS FIRST' });
+                    m.set(key + '_ASC_NULLS_LAST', { [key]: 'ASC NULLS LAST' });
+                    m.set(key + '_DESC_NULLS_FIRST', { [key]: 'DESC NULLS FIRST' });
```

Comment:
Compilation fails, because `OrderBy` type should be adjusted [here](https://github.com/subsquid/squid-sdk/blob/ec9a93fb5a58c7f08c7176106722370d6b5b1849/graphql/openreader/src/ir/args.ts#L85) as well